src/actions/VaccineActions.js

```diff
@@ -38,13 +40,39 @@ const startSensorScan = () => async (dispatch, getState) => {
     return null;
   }
 
-  dispatch(scanForSensors());
+  await BleService()
```

Comment:
I had to go test this, it confused me! the await as well as `.catch()` made me unsure if the `.catch()` worked, but it does .. bit of a nit: either `await` and `try/catch` or `.catch()` with no await?

Reply:
good call - I'll tidy up.